djes/mapping.py

```diff
@@ -5,20 +5,37 @@
 
 from djes.conf import settings
 
-# TODO: expand this for all django field types
 FIELD_MAPPINGS = {
     "AutoField": {"type": "long"},
-    "OneToOneField": {"type": "long"},
-    "ForeignKey": {"type": "long"},
-    "ManyToManyField": {"type": "long"},
-    "IntegerField": {"type": "long"},
+    "BigIntegerField": {"type": "long"},
+    "BinaryField": {"type": "binary"},
+    "BooleanField": {"type": "boolean"},
     "CharField": {"type": "string"},
-    "TextField": {"type": "string"},
-    "SlugField": {"type": "string", "index": "not_analyzed"},
-    "DateTimeField": {"type": "date"},
+    "CommaSeparatedIntegerField": {"type": "string"},
     "DateField": {"type": "date"},
-    "BooleanField": {"type": "boolean"},
+    "DateTimeField": {"type": "date"},
+    "DecimalField": {"type": "string"},
+    "DurationField": {"type": "long"},
+    "EmailField": {"type": "string"},
+    # "FileField": {"type": ""},  # TODO: make a decision on this
+    "FilePathField": {"type": "string"},
+    "FloatField": {"type": "double"},
+    # "ImageField": {"type": ""},  # TODO: make a decision on this
+    "IntegerField": {"type": "number"},
```

Comment:
By default (when using dynamic mappings) ES will choose the safest option - `long`. It can however save you some space on disk and in memory if you know you can map to a smaller numeric type (`integer`, `short` or `byte`) so it might be worth it to be more strict (especially with `SmallIntegerField`)